.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,28 @@
+name: Deployment
```

Comment:
Looks good, don't forget to fix the npm-publish.yml as that is currently broken :)

Reply:
I plan to open a separate PR for that in the future as I believe it is out of scope for this one.